src/main/java/com/example/shorturl/model/user/UserEntity.java

```diff
@@ -0,0 +1,34 @@
+package com.example.shorturl.model.user;
+
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Table(name = "users")
+public class UserEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private int id;
+    private String userName;
+    private String password;
+    private String roles;
```

Comment:
I don't see any relations

Reply:
Should I create another table with different roles?